src/main.rs

```diff
@@ -343,54 +327,21 @@ async fn run_test(test_args: TestArgs) -> anyhow::Result<i32> {
         failures: Vec::new(),
     });
 
-    let quarantine_results = if run_result.failures.is_empty() {
-        QuarantineBulkTestStatus {
-            group_is_quarantined: false,
-            quarantine_results: Vec::new(),
-        }
-    } else {
-        match Retry::spawn(default_delay(), || {
-            get_quarantine_bulk_test_status(
-                &api_address,
-                token,
-                org_url_slug,
-                &repo.repo,
-                &run_result.failures,
-            )
-        })
-        .await
-        {
-            Ok(quarantine_results) => quarantine_results,
-            Err(e) => {
-                log::error!("Failed to get quarantine results: {:?}", e);
-                QuarantineBulkTestStatus {
-                    group_is_quarantined: false,
-                    quarantine_results: Vec::new(),
-                }
-            }
-        }
-    };
-
-    log::info!("Quarantine results: {:?}", quarantine_results);
-    // use the exit code from the command if the group is not quarantined
-    // override exit code to be exit_success if the group is quarantined
-    let exit_code = if *dry_run {
```

Comment:
Where did dry run went?

Reply:
Btw `dry_run` block is a good place to print quarantined tests info.

Reply:
I removed that as initially we were thinking to always upload for quarantining `dry_run`, but will add it back as we are now splitting that logic.